robottelo/hosts.py

```diff
@@ -2166,6 +2166,29 @@ def enroll_ad_and_configure_external_auth(self, ad_data):
             self.execute('systemctl daemon-reload && systemctl restart httpd.service').status == 0
         )
 
+    def configure_libvirt_cr(self, server_fqdn=settings.libvirt.libvirt_hostname):
```

Comment:
Adding helpers directly in host classes is prohibited, please add them to any of the Mixins or create new mixin class and make that a parent of this class.

Reply:
I wouldn't mind adding it to mixins, but I'm not sure since when we're prohibiting the helpers addition to host classes :)

Reply:
Sorry I missed the key work in my initial comment. I mean to say we dont want to add non-host-classes related helpers into that module directly, that way the file would increase into tones. 

Hence the reason we have Mixins !

Reply:
yes, I agree on it @jyejare , so I've already updated the PR to move it into mixins